GeneralsMD/Code/GameEngine/Source/GameClient/MessageStream/SelectionXlat.cpp

```diff
@@ -1099,10 +1100,13 @@ GameMessageDisposition SelectionTranslator::translateGameMessage(const GameMessa
 					m_lastGroupSelTime = now;
 				}
 
+				Bool performSelection = TRUE;
```

Comment:
I suggest add a "TheSuperHackers" comment somewhere here explaining what was changed and why.